securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/InternationalScheduledPaymentConsentDetailsConverterTest.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright © 2020-2021 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.securebanking.openbanking.uk.rcs.converters;
+
+import com.forgerock.securebanking.common.openbanking.uk.forgerock.datamodel.common.FRExchangeRateInformation;
+import com.forgerock.securebanking.openbanking.uk.rcs.api.dto.consent.details.InternationalScheduledPaymentConsentDetails;
+import com.google.gson.JsonObject;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static com.forgerock.securebanking.openbanking.uk.rcs.converters.DomesticScheduledPaymentConsentDetailsConverter.DATE_TIME_FORMATTER;
+import static com.forgerock.securebanking.openbanking.uk.rcs.converters.UtilConverter4Test.INTERNATIONAL_PAYMENT_INTENT_ID;
+import static com.forgerock.securebanking.platform.client.test.support.InternationalScheduledPaymentConsentDetailsTestFactory.aValidInternationalScheduledPaymentConsentDetails;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Unit test for {@link InternationalScheduledPaymentConsentDetailsConverter}
+ */
+@Slf4j
+public class InternationalScheduledPaymentConsentDetailsConverterTest {
+    @Test
+    public void shouldConvertConsentDetailsToInternationalScheduledPaymentConsentDetails() {
+        // Given
+        JsonObject consentDetails = aValidInternationalScheduledPaymentConsentDetails(INTERNATIONAL_PAYMENT_INTENT_ID);
```

Comment:
That `INTERNATIONAL_PAYMENT_INTENT_ID` seems wrong for that case, I expect `INTERNATIONAL_SCHEDULED_PAYMENT_INTENT_ID`, probably is to reuse the value, but is confuse to read the source.